operator/v1/types_network.go

```diff
@@ -349,6 +353,35 @@ type HybridOverlayConfig struct {
 type IPsecConfig struct {
 }
 
+type PolicyAuditConfig struct {
+	// RateLimit is the approximate maximum number of messages to generate per-second per-node. If
```

Comment:
lower-case all field names in the description. They relate to the JSON representation.

Reply:
Sounds good